README.md

```diff
@@ -418,6 +420,79 @@ fga model **validate**
 {"is_valid":false,"error":"the relation type 'employee' on 'member' in object type 'group' is not valid"}
 ```
 
+##### Run tests on an Authorization Model
+
+Given a model, and a set of tests (tuples, check and list objects requests, and expected results) report back on any tests that do not return the same results as expected.
+
+###### Command
+fga model **test**
+
+###### Parameters
+
+* `--tests`: Name of the tests file. Must be in yaml format (see below),
+* `--file`: File containing the authorization model. (optional) [WARNING: If given a model file, this will write it to the store and it will become the latest model.]
```

Comment:
Maybe we can remove this option, it's kinda dangerous. You'd need to write the model first.

Reply:
Yep, we can add it back later once we're doing the full evaluation locally in the CLI

Reply:
Fixed